src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.html

```diff
@@ -1,13 +1,10 @@
-<div class="form-check form-switch">
-    <input
-        class="form-check-input"
-        type="checkbox"
-        id="enabledDisabledSwitch"
-        [disabled]="!isAdmin && settingsType !== EXERCISE"
-        [(ngModel)]="enabled"
-        (change)="onEnabledChange()"
-    />
-    <label class="form-check-label" for="enabledDisabledSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.enabled-disabled"> Enabled/Disabled </label>
+<div class="btn-group">
+    <div class="btn" [ngClass]="{ 'btn-success selected': subSettings!.enabled, 'btn-default': !subSettings!.enabled }" (click)="onEnable()">
```

Comment:
Please do not use `subSettings!` here. In case `subSettings` is undefined for whatever reason (though it should not happen it can happen in rare cases due to Angular weirdness), this would completely crash the view.

Reply:
🙏🏼